base/cvd/cuttlefish/host/commands/cvd/cli/selector/selector.cpp

```diff
@@ -93,17 +128,38 @@ Result<LocalInstanceGroup> PromptUserForGroup(InstanceManager& instance_manager,
   }
 }
 
+Result<LocalInstanceGroup> FindGroupOrDefault(
+    const InstanceDatabase::Filter& filter,
+    const InstanceManager& instance_manager) {
+  if (filter.Empty()) {
+    auto default_group = CF_EXPECT(GetDefaultGroup(instance_manager));
+    return default_group;
+  }
+  auto groups = CF_EXPECT(instance_manager.FindGroups(filter));
+  CF_EXPECT(groups.size() == 1, "groups.size() = " << groups.size());
```

Comment:
should we use ` CF_EXPECT_EQ` to be consistent as in  line 147 with `CF_EXPECT_EQ(instances.size(), 1u, ...`?

Reply:
Done